src/scanners/css.js

```diff
@@ -0,0 +1,67 @@
+import BaseScanner from 'scanners/base';
+import * as cssParser from 'css';
+import * as rules from 'rules/css';
+import { CSS_SYNTAX_ERROR } from 'messages';
+import { VALIDATION_ERROR } from 'const';
+import { ignorePrivateFunctions } from 'utils';
+
+
+export default class CSSScanner extends BaseScanner {
```

Comment:
Shame extending didn't dry things up more for this CSS scanner. I wonder if we could re-think and maybe optionally pre-process the descriptors somehow to avoid repeating most of the scan function.